src/.vuepress/components/SwaggerUI.vue

```diff
@@ -141,6 +154,9 @@
           });
         }
 
+        // Update integer color to meet WCAG color contrast
+        Translate.updateIntegerColor();     
```

Comment:
This is to fix the red text in the code blocks:
<img width="603" height="236" alt="Screenshot 2025-09-09 at 12 00 42 PM" src="https://github.com/user-attachments/assets/e0ffab2f-284b-4129-9288-a262be4c4b36" />

Thanks @andrewleith!